### DISCUSSION TITLE
medicines name is not working in Live Transcribe

### DISCUSSION THREAD

**Discussion by kamlesh0606** [deepgram/community#493] [General help]
Hello,

Currently , we opened new account and check Speech to Text using Live Transcribe. we tried with Medical Text Data But medicines name is not come.

Please provide update on it.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by kamlesh0606** [reply]
Hello,

**Request ID** :: 6d9034e6-b3a0-437f-8118-64c11288e981

**Input Text Details** : "A female patient named Komal Patel, age 32 years, from Ahmedabad. Komal started on aspirin 100mg per day, glopinogrel 75mg per day, heparin 5000mg every 6-8 hours for first 5 days and beta blocker betapolon 300mg per day".

**Outcome at Live Transcribe Demo UI** : "A female Patient name, Kumal Patel, age 32 years from Ahmedabad. So Kumho started on SPIN under mg per day, Blue p No grill, 75 mg per day. Hiperine, 5,000 mg every six 8 hours for 1st 5 days and beta local beta All on 300 mg per day."

**Comment by jkroll-deepgram** [comment]
Hi @kamlesh0606, I recommend trying our medical model, which is trained on more healthcare terminology such as medicine names. You can send your request to `model=nova-2-medical`. We offer the medical model for US English (`language=en-US`) as well as global English (`language=en`). That should generally improve accuracy of your medical transcription.

We also have a [keywords](https://developers.deepgram.com/docs/keywords) feature that improves recognition of words that our model has not seen during training, so it can help in some cases if you have a medicine name or other word that is consistently not recognized. For instance, to recognize "betapolon", you can specify `keywords=betapolon`, or for an even higher intensifier, `keywords=betapolon:2`. That should improve targeted accuracy of specific challenging out-of-vocabulary words.

**Reply by kamlesh0606** [reply]
@jkroll-deepgram  Please Demo link if there.
i checked website bit did not get demo link for model=nova-2-medical .

**Reply by jkroll-deepgram** [reply]
Hi @kamlesh0606, our Playground demo does not currently support the medical model. Depending on whether you're working with pre-recorded or streaming audio, you can check out our quickstarts for each: [pre-recorded](https://developers.deepgram.com/docs/getting-started-with-pre-recorded-audio), [streaming](https://developers.deepgram.com/docs/getting-started-with-live-streaming-audio). Those tutorials use the `nova-2` model, and you would change it to be `nova-2-medical`. You can make that same change in any other of our samples or code that you write yourself.
